### PR TITLE
Ahayel/email sending

### PR DESCRIPTION
**Pull request by artemgael96**

### CODE REVIEW

File: exalearn-api/ExaLearn.Bl/Services/TestService.cs [hunk 1]
```diff
@@ -69,6 +73,9 @@ public async Task<AssessmentDTO> CreateAssesmentAsync(AssessmentDTO assessmentDT
 
             passedTest.Status = StatusType.Checked;
 
+            var user = await _userRepository.GetByIdAsync(passedTest.UserId);
+
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Bl/Services/TestService.cs:84]: var m = MessageBuilder.GenerateMessageForUserAsync(user, levelType, expirationDate, isAssing);

File: exalearn-api/ExaLearn.Bl/Services/TestService.cs [hunk 2]
```diff
@@ -69,6 +73,9 @@ public async Task<AssessmentDTO> CreateAssesmentAsync(AssessmentDTO assessmentDT
...
+            EmailSender.SendEmailAsync(user, passedTest.LevelType, passedTest.PassedTestDate, false).GetAwaiter();
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Bl/Services/TestService.cs]: create model in params

File: exalearn-api/ExaLearn.Bl/EmailService/MessageBuilder.cs [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            LevelType levelType,
+            DateTime expirationDate,
+            string fullName,
+            bool isAssing
+            )
+        {
+            var message = new MailMessage(from, to)
+            {
+                Subject = isAssing ? "You are assigned to a test" : "Your a test results",
+                Body = isAssing
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Bl/EmailService/MessageBuilder.cs]: EmailBodyGenerator with 2 method return template and 1 method with switch

**Review by vlad12345123 — APPROVED** [review]

File: exalearn-api/ExaLearn.Bl/Services/TestService.cs (hunk 1 above)
Comment (artemgael96) [reply exalearn-api/ExaLearn.Bl/Services/TestService.cs:84]: done

File: exalearn-api/ExaLearn.Bl/EmailService/MessageBuilder.cs (hunk 3 above)
Comment (artemgael96) [reply exalearn-api/ExaLearn.Bl/EmailService/MessageBuilder.cs]: done

**Review by Atalay-Polat — APPROVED** [review]

**Pull request merged**